src/script/main/app.ts

```diff
@@ -815,6 +815,7 @@ class App {
         }
       }
 
+      await this.core.logout(clearData);
```

Comment:
should this be conditional to MLS?

Reply:
The core is the one knowing what to do on logout (deleting database for example). But the webapp needs to call `logout` everytime someone logs out (proteus as well)

Reply:
ah sorry didnt read the other PR yet, seems fine